### DISCUSSION TITLE
Docker image registry for Chemprop 2+

### DISCUSSION THREAD

**Discussion by xuyan-ru** [chemprop/chemprop#838] [General]
Hi guys, 

thanks for releasing the chemprop 2.0, just would like to know is there any update on using chemprop as a docker iamge, is it already registered in gihtub registry or dockerhub now or it will be released in a following patch or feature release?

Thanks!

**Comment by JacksonBurns** [comment]
Thanks for the reminder - doing this now.

**Reply by JacksonBurns** [reply]
Docker image is now up for v2, nightly builds will be available shortly.

**Comment by kevingreenman** [comment]
see #841

**Reply by xuyan-ru** [reply]
Thanks Kevin! Looking forward to it
